### ISSUE TITLE
Reversed log level order 

### ISSUE THREAD

**Issue by akurnaz** [adaptyteam/AdaptySDK-Flutter#73]
When I set the log level to "debug", I expect to see other levels log. Similarly, for "error" level, I expect the only errors logs. But now the opposite is happening.

sdkVersion = '2.2.3'

**Comment by x401om** [comment]
@akurnaz I don't fully understand the problem. In fact, the `debug` level was created for internal use, but it is more extensive than `verbose`. So, if you make `logLevel = debug`, you should see everything included in verbose.
If you think it actually works differently in the real world, could you please attach the full console log, please?

**Comment by akurnaz** [comment]
@x401om Sorry for explaining it complicated. I am adding an example.

When log level is debug:

```
Adapty().setLogLevel(AdaptyLogLevel.debug);
      
AdaptyLogger.write(AdaptyLogLevel.debug, 'debug');
AdaptyLogger.write(AdaptyLogLevel.verbose, 'verbose');
AdaptyLogger.write(AdaptyLogLevel.info, 'info');
AdaptyLogger.write(AdaptyLogLevel.warn, 'warn');
AdaptyLogger.write(AdaptyLogLevel.error, 'error');
```

console logs:

> flutter: [AdaptyFlutter v2.2.4] - DEBUG: debug

When log level is error:

```
Adapty().setLogLevel(AdaptyLogLevel.error);

AdaptyLogger.write(AdaptyLogLevel.debug, 'debug');
AdaptyLogger.write(AdaptyLogLevel.verbose, 'verbose');
AdaptyLogger.write(AdaptyLogLevel.info, 'info');
AdaptyLogger.write(AdaptyLogLevel.warn, 'warn');
AdaptyLogger.write(AdaptyLogLevel.error, 'error');
```

console logs:

> flutter: [AdaptyFlutter v2.2.4] - VERBOSE: --> Adapty.set_log_level()
> flutter: [AdaptyFlutter v2.2.4] - DEBUG: debug
> flutter: [AdaptyFlutter v2.2.4] - VERBOSE: verbose
> flutter: [AdaptyFlutter v2.2.4] - INFO: info
> flutter: [AdaptyFlutter v2.2.4] - WARN: warn
> flutter: [AdaptyFlutter v2.2.4] - ERROR: error

**Comment by x401om** [comment]
@akurnaz it was a bug in the flutter plugin, logging in iOS and Android SDKs are fine.
I have fixed the bug in the development branch, so it will go in the next release (It is scheduled for the next week). Thank you for the report!

**Comment by x401om** [comment]
It was finally fixed in [2.4.0](https://github.com/adaptyteam/AdaptySDK-Flutter/releases/tag/2.4.0)!
